scalafix/rules/src/main/scala/fix/v0_14_0/FixAvroCoder.scala

```diff
@@ -128,6 +133,19 @@ class FixAvroCoder extends SemanticRule("FixAvroCoder") {
           // Coder[T] is a variable type where T is an avro type
           findMatchingValTypeParams(t, CoderMatcher)
             .exists(isAvroType)
+        case q"$jobTestBuilder(..$args)" if JobTestBuilderMatcher.matches(jobTestBuilder) =>
```

Comment:
I spent some time trying to make this generic, so that it would match any function with an implicit coder evidence `T`, by inspecting each function argument for one that matched type `T`, but it got pretty complex/edge-casey. Not sure it's worth it